### DISCUSSION TITLE
[three] Material type became read-only

### DISCUSSION THREAD

**Discussion by MarvinKubus** [DefinitelyTyped/DefinitelyTyped#71415] [Issues with a @types package]
I have an extended material that sets the `type`, just like the example of https://github.com/mrdoob/three.js/blob/dev/src/materials/LineBasicMaterial.js does.
This worked great, but with the recent PR of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71147 it is now giving an error on `type` being read-only.
Should the `type` be set differently (and the example updated), or should it not be read-only?

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Hi @MarvinKubus, do you mind creating this issue over at https://github.com/three-types/three-ts-types? That's where the three.js types are maintained and I'm more likely to take the time to respond over there.

**Reply by MarvinKubus** [reply]
Sure, will do that and close this ticket.
